doc/using/InvokingGHDL.rst

```diff
@@ -233,17 +233,17 @@ Options
 
   Use option :option:`-P` to specify where libraries other than ``WORK`` are placed.
 
-.. option:: --std=<STD>
+.. option:: --std=<STANDARD>
 
-  Specify the standard to use. By default, the standard is ``93c``, which means VHDL-93 accepting VHDL-87 syntax. For details on ``STD`` values see section :ref:`VHDL_standards`.
+  Specify the standard to use. By default, the standard is ``93c``, which means VHDL-93 accepting VHDL-87 syntax. For details on ``STANDARD`` values see section :ref:`VHDL_standards`.
 
-.. option:: --ieee=<VER>
+.. option:: --ieee=<IEEE_VER>
```

Comment:
Maybe `IEEE_VAR` (from *ieee variant*) can be easier to understand than `IEEE_VER` (from *ieee version*)? Both of them are ok. It is up to you.